term_posix.go

```diff
@@ -195,3 +195,8 @@ func (t *Term) Close() error {
 	t.fd = -1
 	return err
 }
+
+// GetFd get terminal's file descripter
+func (t *Term) GetFd() int {
```

Comment:
It seems odd that this function signature would not match up with `os.File.Fd` https://golang.org/pkg/os/#File.Fd